src/polynomial.jl

```diff
@@ -0,0 +1,49 @@
+# function Polynomial{C,E}(vec::AbstractVector, mat::AbstractMatrix) where {C <: Union{CxxWrap.CxxLong,VecOrMat_eltypes}, E <: Union{CxxWrap.CxxLong, Int64}}
+#     v = Vector{C}(vec)
+#     m = Matrix{E}(mat)
+#     return Polynomial{to_cxx_type(C),to_cxx_type(E)}(v, m)
+# end
+#
+# function Polynomial{C,E}(p::Polynomial) where {C,E}
+#     v = coefficients_as_vector(p)
+#     m = monomials_as_matrix(p)
+#     return Polynomial{C,E}(v, m)
+# end
+#
+# function Polynomial{C,E}(n::Number, d::Base.Integer) where {C,E}
+#     m = zeros(1,d)
```

Comment:
this is Base.Matrix of Float64s, use `Matrix{E}(under, 1,d)` (our matrices are 0-initialized)